pkg/services/ngalert/metrics/ngalert.go

```diff
@@ -183,6 +184,15 @@ func newSchedulerMetrics(r prometheus.Registerer) *Scheduler {
 			},
 		),
 		Ticker: legacyMetrics.NewTickerMetrics(r),
+		DroppedTicksTotal: promauto.With(r).NewCounterVec(
+			prometheus.CounterOpts{
+				Namespace: Namespace,
+				Subsystem: Subsystem,
+				Name:      "rule_dropped_evaluations",
+				Help:      "The total number of evaluations dropped for a rule.",
+			},
+			[]string{"org", "rule_uid"},
```

Comment:
> this would produce less series.

not sure I understand how it would produce less series. The title of alert can change but UID of alert cannot

Reply:
Sorry, I was thinking of groups as opposed to individual rules you're right that this wouldn't produce fewer series.

My motivation for the rule name is that it's easily identifiable in metrics resorting to UIDs is not obvious without looking them up.

Reply:
name can change for the same alert rule. What do you think about having both labels? Also, the rule name is not in the context of where that metric is maintained. So I will have to update ShedulableAlertRule to fetch title from database.

Reply:
Having two labels is fine, but I'd drop the rule_ prefix -- the metric is already called rule_evaluations_missed_total and I just realised that it needs to be prefixed with schedule because that's the area name so that it matches others -- sorry about that, I must have missed it. So for name:
- `schedule_rule_evaluations_missed_total`
- and for labels `uid` and `name`

I'll say again that having IDs as label values is basically an antipattern (see https://prometheus.io/docs/practices/naming/#labels) -- you can create a rule with a name then delete and create it again with the same name - you'll have two series stored as part of that as opposed to just 1 if you use the name. That being said, If I really had to be dogmatic about it -- the right combination of labels is probably folder, group name and lastly rule name. So for now, I'd probably choose folder and rule name. With group added once you complete the rules sequential evaluation. 

But, I think that once we switch to groups being the top-level item all of this will have to change and it won't matter as much -- so... I'm fine if you want to proceed with `uid` and name for `name` as we can correct later (and probably end up breaking people's dashboards/alerts again).